source/common/protobuf/utility.h

```diff
@@ -366,9 +366,10 @@ class MessageUtil {
   /**
    * Modifies a message such that all sensitive data (that is, fields annotated as
    * `udpa.annotations.sensitive`) is redacted for display. String-typed fields annotated as
-   * `sensitive` will be replaced with the string "[redacted]", primitive-typed fields (including
-   * enums) will be cleared, and message-typed fields will be traversed recursively to redact
-   * their contents.
+   * `sensitive` will be replaced with the string "[redacted]", bytes-typed fields will be replaced
```

Comment:
+1 this is much cleaner than before's DataSource type switch case handling.

LGTM.